phpunit-integration.xml

```diff
@@ -1,17 +1,18 @@
-<phpunit
-        bootstrap="tests/bootstrap.php"
-        colors="true"
-        convertNoticesToExceptions="true"
```

Comment:
the removed ones either no longer exist or are defaults in new version of PHPUnit